packages/core/src/types/redocly-yaml.ts

```diff
@@ -183,29 +185,43 @@ type BuiltInRuleId = typeof builtInRules[number];
 const ConfigStyleguide: NodeType = {
   properties: {
     extends: {
-      type: 'array',
-      items: {
-        type: 'string',
+      name: 'ConfigStyleguideList',
```

Comment:
Renamed to `ConfigGovernance` [here](https://github.com/Redocly/redocly-cli/pull/2131/files#diff-2528117ea3cbe25dfbfd0583c9b11ba7baa7ef347184dfba32a73ec486aa1a5e).